chips/stm32f429zi/src/lib.rs

```diff
@@ -1,7 +1,109 @@
 #![no_std]
 
-pub use stm32f4xx;
+use cortexm4::generic_isr;
+
 pub use stm32f4xx::{chip, dbg, dma1, exti, gpio, nvic, rcc, spi, syscfg, tim2, usart};
 
-pub mod irqs;
 pub mod stm32f429zi_nvic;
+
+// STM32F42xxx and STM32F43xxx has total of 91 interrupts
+#[cfg_attr(all(target_arch = "arm", target_os = "none"), link_section = ".irqs")]
+// used Ensures that the symbol is kept until the final binary
```

Comment:
```suggestion
// `used` ensures that the symbol is kept until the final binary. However, as of
// May 2020, due to the compilation process, there must be some other compiled
// code here to make sure the object file is kept around. That means at minimum
// there must be an `init()` function here so that compiler does not just ignore
// the `IRQS` object. See https://github.com/rust-lang/rust/issues/56639 for a
// related discussion.
```